### ISSUE TITLE
Inquiry Regarding Reproducing Results in Table 1 of "The Open Proof Corpus" Paper

### ISSUE THREAD

**Issue by ArthurZhang02** [insait-institute/open-proof-corpus#3]
Dear Authors,

Thank you for your fascinating work on "The Open Proof Corpus: A Large-Scale Study of LLM-Generated Mathematical Proofs." I am writing to ask for a clarification to help in reproducing the results presented in Table 1 (Benchmarking LLMs as proof graders).

To ensure I use the correct dataset, could you please specify whether the evaluation was performed on the `hf-opc-test dataset` (with 292 rows) available on Hugging Face, or on the `test.parquet` file (with 434 rows) included in your reinforcement learning code repository?

Any guidance you could provide would be greatly appreciated.

**Comment by JasperDekoninck** [comment]
Thanks for the question!
The evaluation was performed with the dataset on HuggingFace. test.parquet contains the validation set that was used when training the model and is completely separate from the actual test set.

**Comment by ArthurZhang02** [comment]
Thanks for your reply!

I also have a question regarding the training data used in the VERL framework. I noticed that the `train.parquet` file contains only about 1.7k samples, whereas the “generic” dataset on Hugging Face has around 3k samples. Could you help clarify why there is this difference in the number of samples?(Did you use some filtering technique in data process?)

Thank you in advance for your help!

**Comment by ArthurZhang02** [comment]
By the way, could you please provide the parameters for testing LLM on `test` set, such as temperature and other information?

**Comment by 1414vo** [comment]
We indeed used a filtering process in for deriving the train and validation (`test.parquet`) sets. We removed any solutions that were missing text feedback, or contained a disagreement between the 2 judges on the problem. We then held out the solutions produced by DeepSeek R1, and split the remaining dataset into training and validation on a per-problem basis. Further, the training and validation sets were fixed before we finalized the "generic" portion of our dataset, so there will likely be some problems that would pass the filtering criteria that do not appear in the parquet files.

For testing the LLMs on the LLM-as-a-judge task, we used the configurations in `configs/models`, where if a parameter is not overridden, the parameters default to the ones given in `configs/setting` configuration.
